src/components.d.ts

```diff
@@ -531,6 +531,20 @@ export namespace Components {
          */
         "isclosable": boolean;
     }
+    interface ZInfoReveal {
+        /**
+          * Name of the icon for the open button
+         */
+        "icon"?: string;
+        /**
+          * Text that appears on closed panel next to the open button.
+         */
+        "label"?: string;
+        /**
+          * Position of the open/close button on the panel.
```

Comment:
mancherebbe sempre in entrambi e soprattutto in index.tsx la descrizione dei possibili valori. Con la modifica dell'altro parametro è sparito il commento ^^

Reply:
Con typescript abbiamo la possibilità, tramite l'IDE, di vedere i valori possibili per una prop (anche quando è un enum, ad esempio). Basta soffermarsi un attimo sul nome della prop, nel file .tsx, e si ottiene il suggerimento.

Ho adeguato anche altri componenti per far funzionare questa "feature" (bisognava dare la stessa formattazione maiusc/minusc a prop/valori per far sì che funzionasse).

Mi sembra un buon compromesso che non introduce breaking. 

metto in cc @DarioSacco che ha proposto la cosa

Reply:
per curiosità, che IDE usate/con che estensione? Perché io con l'hover sulla prop nel .tsx vedo solo riproposta la descrizione ^^'

Reply:
Scusa, mancava da cambiare la definizione della prop, in questo modo dovresti vederlo (vedi ultimo commit).